joomla/joomla-cms
# [4.0] com_privacy consents

### Summary of Changes
Updates the tmpl for the consents view to match J4 standards and styles
Add name column
Add search by name filter
Rename "Created" column header to "Consented"

### Testing Instructions
To test it will be easier if you have some data. These three rows should cover all the options - obviously replace jos_ with your own db prefix AND user_id with a valid id
```
INSERT INTO `jos_privacy_consents` (`id`, `user_id`, `state`, `created`, `subject`, `body`, `remind`, `token`) VALUES (1, 540, 1, '2019-07-01 19:36:00', 'title', 'body', 0, '');
INSERT INTO `jos_privacy_consents` (`id`, `user_id`, `state`, `created`, `subject`, `body`, `remind`, `token`) VALUES (2, 540, 0, '2018-06-01 12:01:00', 'title', 'body', 0, '');
INSERT INTO `jos_privacy_consents` (`id`, `user_id`, `state`, `created`, `subject`, `body`, `remind`, `token`) VALUES (3, 540, -1, '2019-07-01 07:16:00', 'title', 'body', 0, '');

```

### Expected result
![image](https://user-images.githubusercontent.com/1296369/62109821-02edd600-b2a5-11e9-9e7f-650c84fe7955.png)


Other parts of com_privacy are beyond the scope of this PR
